.travis.yml

```diff
@@ -52,6 +55,6 @@ jobs:
     # must repeat the s390x job above exactly to match
     - *linux_s390x_36
 
-install: $PYTHON -m pip install -r requirements-dev.txt
+install: $PYTHON -m pip install -r requirements-dev.txt pytest-custom-exit-code
```

Comment:
how this is used?

Reply:
In the piece of code of the previous remark you made: `PYTEST_ADDOPTS='-k "unit_test or test_0_basic" --suppress-no-test-exit-code'`. It's a plugin, not standard included in pytest.

AppVeyor does something similar. If I remember correctly, it's because pytest fails (or warns; I can't remember) and exits with a non-zero exit-code if no tests at all are selected. But since all our tests are separate calls to `pytest` in `bin/run_tests.py`, `-k "unit_test or test_0_basic"` will make some of those empty, and we need to ignore that to have everything succeed.